index.html

```diff
@@ -1501,6 +1503,7 @@
 
 </UL>
 
+<!-- Modern tools don't need this kind of help. Thank goodness.
```

Comment:
Trying to document tools not even knowing what people are using was always a bit tricky, but what we have here is out of date enough that getting rid of it seems like an improvement.